testsuites/CBLTester/topology_specific_tests/multiple_sync_gateways/conftest.py

```diff
@@ -69,6 +69,9 @@ def params_from_base_suite_setup(request):
     disable_tls_server = request.config.getoption("--disable-tls-server")
 
     disable_admin_auth = request.config.getoption("--disable-admin-auth")
+    print("==============================================")
```

Comment:
clean up prints

Reply:
cleaned up the print statement, and added proper log_info